packages/core/plugin/src/PluginAPI.js

```diff
@@ -12,7 +12,7 @@ import type {
   Validator as ValidatorOpts
 } from '@parcel/types';
 
-export const CONFIG = Symbol('config');
+export const CONFIG = Symbol.for('parcel-plugin-config');
```

Comment:
I've updated this to `Symbol.for('parcel-plugin-config');` so we definitely won't have any conflicts as using a global key `config` seems too likely to already exist to me